### ISSUE TITLE
Add ability to declare queue

### ISSUE THREAD

**Issue by NicholasAStuart** [pmq/rabbitmq-oracle-stored-procedures#8]
You have the ability to declare exchanges, but not the ability to decare queue's. Please add this feature.

**Comment by pmq** [comment]
Hi Nicholas, yes I can add this feature. But in most scenarii wouldn't you probably prefer to declare queues from your consumers? Let me know.

**Comment by NicholasAStuart** [comment]
Yes, however I may be pushing code before my consumer has come online and I'd like for the producer to be able to declare the queue in case it doesn't already exist so no messages are lost.

**Comment by pmq** [comment]
Hi Nicholas, just to let you know I'll probably be able to push this feature today or tomorrow.

**Comment by AshutoshPanda** [comment]
is it possiable to write a trigger which can send a message to rabbitmq for DML(insert)  mysql table

**Comment by kimoturbo** [comment]
Regarding the ability to declare queues, did you add this feature or not yet? If so, can you add it and update the program?
